### PR TITLE
content inside card centered

### PR DESCRIPTION
**Pull request by rahulrana701**
I have centered the content inside the cards as you suggested and applied the screenshots you told me to 
Screenshot 2023-06-04 233808
<br>
Screenshot 2023-06-04 233642

### CODE REVIEW

**Comment by rahulrana701** [comment]
hi @JasonDsouza212 I closed the earlier pr and created a new pr for the issue and this pr have no conflicts , sorry for the inconvenience..

**Comment by rahulrana701** [comment]
@JasonDsouza212 can you please review it

**Review by VenomFate-619 — APPROVED** [review]

**Comment by rahulrana701** [comment]
hi @VenomFate-619 what will happen to this pr , because you have assigned the same issue to someone else

**Comment by JasonDsouza212** [comment]
Always add the issue you are solving

**Comment by rahulrana701** [comment]
@JasonDsouza212 merge conflicts resolved..

File: src/styles/Home.css [hunk 1]
```diff
@@ -81,14 +81,10 @@ img {
   margin-top: 2rem;
 }
 
-.grid .text {
```
Comment (rahulrana701) [inline src/styles/Home.css]: .grid .text

**Pull request closed without merging**